components/tidb_query_vec_expr/src/types/expr.rs

```diff
@@ -2,11 +2,39 @@
 
 use std::any::Any;
 
-use tipb::FieldType;
+use tipb::{ExprType, FieldType};
 
 use super::super::function::RpnFnMeta;
 use tidb_query_datatype::codec::data_type::ScalarValue;
 
+#[derive(Debug, Clone, Copy)]
+pub enum Operator {
```

Comment:
What's the purpose of introducing a new `Operator` type?

Reply:
> What's the purpose of introducing a new `Operator` type?

If there is no Operator type, rpn-expression can't express a full Aggregation.